way_to_home/user_profile/models.py

```diff
@@ -1 +1,53 @@
 """This module implements class that represents the user profile entity."""
+
+from user.models import User
+from django.db import models, IntegrityError
+from utils.abstract_models import AbstractModel
+
+
+class UserProfile(AbstractModel):
+    """Model for user profile entity."""
+
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    first_name = models.CharField(max_length=64)
+    last_name = models.CharField(max_length=64)
```

Comment:
```suggestion
    first_name = models.CharField(max_length=64)
    last_name = models.CharField(max_length=64)
```

I think it can be blank. Set `blank=True`

Reply:
Done